### ISSUE TITLE
404 not found on WordPress Updates page

### ISSUE THREAD

**Issue by kakshak** [YahnisElsts/plugin-update-checker#380]
Hi there,

Thanks for this plugin. It helped me to provide the plugin updates.

I just want to know one thing that, when I go to the WordPress Updates (update-core.php) page, it gives me a 404 error for v1.2 tag.
Screenshot: https://prnt.sc/tnf7lm

Please help me with this.

**Comment by YahnisElsts** [comment]
Does your repository actually contain a tag named `v1.2` or is the update checker trying to load a non-existent tag for some reason? Also, what is the request just above the one that triggered a 404 error?

**Comment by kakshak** [comment]
Yes, the README.txt file contains `Stable tag: v1.2` tag name (As my current plugin version is also 1.2).

Here, is the request just above the one that triggered a 404:
![image](https://user-images.githubusercontent.com/26460087/88518449-23628480-d00e-11ea-9e07-c43cb97a9d26.png)

**Comment by YahnisElsts** [comment]
That makes sense, the update checker will try to retrieve the stable tag if it's specified. And is there actually a tag named `v1.2`? Note that `v1.2` and `1.2` are not the same when it comes to tag names.

Assuming that the tag actually exists, the only reason I can think of why it would produce a 404 is the authentication default being incorrect, but the other requests work fine so that can't be it.

**Comment by kakshak** [comment]
Do you mean should I update `Stable tag: v1.2` from `Stable tag: 1.2`?

**Comment by YahnisElsts** [comment]
The value of `Stable tag` should be exactly the same as the name of the tag. If the tag is called `1.2`, you should change readme.txt to say `Stable tag: 1.2`

**Comment by gerardpp** [comment]
Sorry, did it fix your problem? I am having the 404 error, but both my tag and stable tags params are 1.0.0.

**Comment by YahnisElsts** [comment]
The issue discussed here only applies to BitBucket. GitHub has a different API.
